src/client/client.js

```diff
@@ -14,6 +14,26 @@ function removeEmpty(obj) {
     return obj;
 }
 
+/**
+ * getAccceptFormat returns the correct Accept header depending on the
+ * requested format.
+ * @param query
+ * @returns {string}
+ */
+function getAccceptFormat(query) {
+    if (typeof query !== "undefined" && query.hasOwnProperty('format'))
```

Comment:
please be consistent with quotes, here are both single and double quotes in the same like. Plus, it this context there is no difference between `typeof query !== "undefined"` and `query !== undefined` since it is declared.

Reply:
You're totally right. Changed them in the latest commit